pkg/invariants/network/disruptionpodnetwork/pod-network-target-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pod-network-disruption-target
+spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 34%
+      maxSurge: 0
+  # to be overridden by the number of nodes
+  replicas: 1
+  selector:
+    matchLabels:
+      network.openshift.io/disruption-target: pod-network
+      network.openshift.io/disruption-actor: target
+  template:
+    metadata:
+      labels:
+        network.openshift.io/disruption-target: pod-network
+        network.openshift.io/disruption-actor: target
+    spec:
+      containers:
+        - command:
+            - /agnhost
+            - netexec
+            - --http-port=8080
+            - --delay-shutdown=30
+          image: registry.k8s.io/e2e-test-images/agnhost:2.43
```

Comment:
Should it be mentioned that the image gets overridden as we do above for replicas?

Reply:
Yeah do a followup